Makefile

```diff
@@ -17,25 +41,19 @@ deploy:
 
 .PHONY:
 what-version:
-	@echo $(VER)
+	@echo $(VERSION)
```

Comment:
I know it's not your code, but we'd be more consistent with `make get-version` (as in other projects)